staff/francesc-destruels/array-functions/expect-1.0.2.js

```diff
@@ -7,7 +7,7 @@ console.log('%c expect', 'font-weight: bold; font-size: 50px; color: red; text-s
  * 
  * @param {*} actual The actual value to check agains the expected value.
  * @param {*} expected The expected value.
- * @param {boolean} [deep] Indicates wether the interal check comparison must be deep or not (in case of arrays or objects).
```

Comment:
@FranDRoss you are updating a file from Manu. Please update PR where only your files are updated, added or removed. You have also renamed two of his files.

Reply:
I will do it ASAP